tests/BreadcrumbTest.php

```diff
@@ -102,4 +102,26 @@ public function testWithoutMetadata(): void
         $this->assertSame(['foo' => 'bar'], $breadcrumb->getMetadata());
         $this->assertArrayNotHasKey('foo', $newBreadcrumb->getMetadata());
     }
+
+    public function testJsonSerialize(): void
+    {
+        $type = Breadcrumb::TYPE_USER;
+        $level = Breadcrumb::LEVEL_INFO;
+        $category = 'foo';
+        $data = ['baz' => 'bar'];
+        $message = 'message';
+        $breadcrumb = new Breadcrumb($level, $type, $category, $message, $data);
+        $breadcrumbSerialized = $breadcrumb->jsonSerialize();
+
+        $expected = [
+            'type' => $type,
+            'category' => $category,
+            'message' => $message,
+            'level' => $level,
+            'timestamp' => microtime(true),
+            'data' => $data
+        ];
+        
+        $this->assertEquals($expected, $breadcrumbSerialized);
```

Comment:
Since we don't use the `$breadcrumbSerialized` variable in more places we can inline the call to the `jsonSerialize` function here. Also build is failing due to CS issues